apps/storefront/src/shared/service/b2b/graphql/orders.ts

```diff
@@ -467,3 +515,11 @@ export const getOrdersCreatedByUser = (companyId: number, module: number) =>
   B3Request.graphqlB2B({
     query: getCreatedByUser(companyId, module, 'createdByUser'),
   });
+
+export const getDigitalDownloadElements = (orderId: number | string) =>
```

Comment:
Let's move this fetch inside the `OrderDetail/components/getDigitalDownloadElements.ts` folder, ideally each component should query the data it needs, without sharing the queries, it avoids bloated queries.